tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/GitHubEventClient.cs

```diff
@@ -940,5 +945,32 @@ public virtual async Task<List<string>> QueryAILabelService(IssueEventGitHubPayl
             }
             return returnList;
         }
+
+        /// <summary>
+        /// Create a raw github URL (https://raw.githubusercontent.com) for a file in a given repository
+        /// </summary>
+        /// <param name="repository">Octkit.Repository from the event payload</param>
+        /// <param name="subdirectory">Subdirectory where the file lives</param>
+        /// <param name="fileName">name of the file</param>
+        /// <returns></returns>
+        public string CreateRawGitHubURLForFile(Repository repository, string subdirectory, string fileName)
+        {
+            // https://raw.githubusercontent.com/Azure/azure-sdk-for-net/main/.github/
+            // The Full URL is BaseUrl + repositoryFullName + defaultBranch + remoteFilePath + fileName
+            string fileUrl = $"{ConfigConstants.RawGitHubUserContentUrl}/{repository.FullName}/{ConfigConstants.DefaultBranch}/{subdirectory}/{fileName}";
+            return fileUrl;
+        }
+
+        /// <summary>
+        /// Set the config file overrides for codeowners and rulesconfig which will cause them to get pulled
+        /// from the URL instead of requiring a sparse checkout of the configuration directory in order to run.
+        /// </summary>
+        /// <param name="repository">Octkit.Repository from the event payload</param>
+        public void SetConfigEntryOverrides(Repository repository)
+        {
+            CodeOwnerUtils.codeOwnersFilePathOverride = CreateRawGitHubURLForFile(repository, CodeOwnerUtils.CodeownersSubDirectory, CodeOwnerUtils.CodeownersFileName);
```

Comment:
Just something to keep in mind this potentially adds more race conditions since these are global static values.